### PR TITLE
[Dust apps cleanup] Remove execute tab from dust apps

### PR DESCRIPTION
**Pull request by albandum**
## Description

Removing 'Run' tab from the Dust apps UI and all related components as execute is pretty much a doublon of the Run button from the specs, and widely less used. 

Screenshot 2024-05-02 at 14 29 15

## Risk
The few people using execute might ask where it's gone. 
But are the few people @spolu and @flvndvd ?

### CODE REVIEW

File: front/pages/api/w/[wId]/apps/[aId]/runs/index.ts [hunk 1]
```diff
@@ -79,89 +79,6 @@ async function handler(
       ]);
 
       switch (req.body.mode) {
```
Comment (spolu) [inline front/pages/api/w/[wId]/apps/[aId]/runs/index.ts:81]: we can remove the body.mode entirely no?

**Review by spolu — COMMENTED** [review]
Works for me. RIP @fontanierh first PR.

Left a small comment

**Comment by fontanierh** [comment]
😭

File: front/pages/api/w/[wId]/apps/[aId]/runs/index.ts (hunk 1 above)
Comment (albandum) [reply front/pages/api/w/[wId]/apps/[aId]/runs/index.ts]: Done

**Review by spolu — CHANGES_REQUESTED** [review]
We likely want to remove the `mode` argument from the run code client side?

**Review by spolu — APPROVED** [review]

**Pull request merged**